include/rocksdb/cache.h

```diff
@@ -127,6 +127,9 @@ extern std::shared_ptr<Cache> NewLRUCache(
 
 extern std::shared_ptr<Cache> NewLRUCache(const LRUCacheOptions& cache_opts);
 
+extern std::shared_ptr<Cache> NewChargedCache(
+    std::shared_ptr<Cache> cache, std::shared_ptr<Cache> block_cache);
+
```

Comment:
This is now redundant, right? That is, we don't need to expose the concept of a charged cache in the public API, and `BlobSource` can simply directly create a charged cache using `make_shared`

Reply:
Done